README.md

```diff
@@ -116,6 +115,8 @@ I would like to thank Microverse for giving me all the necessary knowledge I nee
 
```

Comment:
- [ ] It would be nice if you fix the LinkedIn link in your readme file because it is broken 😨 Kindly fix it to make your readme descriptive and professional 👍

Reply:
It's fixed. Thank you